### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by lex111**
Varios docs improvements.

P.S. Have I made changes to the proper branch?

### CODE REVIEW

File: client-generator/nextjs.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Create a [Next.js application with express server](https://github.com/zeit/next.
 
 ### Installing the Generator Dependencies
 
-Enable Typescript in your next project
+Enable TypeScript in your next project
```
Comment (dunglas) [inline client-generator/nextjs.md]: According to https://www.typescriptlang.org/, the current case is correct.

File: extra/troubleshooting.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The JMS Serializer service is available as `jms_serializer`.
 
 Some of your API calls fail with a 502 error and the logs for the api container shows the following error message `upstream sent too big header while reading response header from upstream`. 
 
-This can be due to the cache invalidation headers that are too big for NGINX. When you query the API, API Platform adds the ids of all returned entities and their dependencies in the headers like so : `Cache-Tags: /entity/1,/dependent_entity/1,/entity/2`. This can overflow the default header size (4k) when your API gets larger and more complex.
+This can be due to the cache invalidation headers that are too big for Nginx. When you query the API, API Platform adds the ids of all returned entities and their dependencies in the headers like so : `Cache-Tags: /entity/1,/dependent_entity/1,/entity/2`. This can overflow the default header size (4k) when your API gets larger and more complex.
```
Comment (dunglas) [inline extra/troubleshooting.md]: According to https://www.nginx.com/, the current case is correct.

Comment (lex111) [reply extra/troubleshooting.md]: Oopsy, you are right, changed!

**Comment by alanpoulain** [comment]
Thank you @lex111.

**Pull request merged**